### PR TITLE
Keep monitoring Active Alerts list ordered by description

### PR DESCRIPTION
**Pull request by kyoto**

### CODE REVIEW

**Comment by kyoto** [comment]
jenkins rebuild

File: frontend/public/components/alert.tsx [hunk 1]
```diff
@@ -229,19 +234,15 @@ const ActiveAlerts = ({alerts}) => <div className="co-m-table-grid co-m-table-gr
...
-        <div className="col-xs-6 co-resource-link-wrapper">
-          <Cog options={[silenceAction(a)]} />
-          <Link className="co-resource-link" to={detailsURL(AlertResource, name, a.labels)}>{description}</Link>
-        </div>
-        <div className="col-xs-2"><Timestamp timestamp={a.activeAt} /></div>
-        <div className="col-xs-2"><State state={a.state} /></div>
-        <div className="col-xs-2">{a.value}</div>
-      </ResourceRow>;
-    })}
+    {_.sortBy(alerts, alertDescription).map((a, i) => <ResourceRow key={i} obj={a}>
```
Comment (spadgett) [inline frontend/public/components/alert.tsx]: Maybe just

```suggestion
    {_.sortBy(alerts, ['annotations.description', 'annotations.message', 'labels.alertname']).map((a, i) => <ResourceRow key={i} obj={a}>
```

Comment (kyoto) [reply frontend/public/components/alert.tsx]: I'm actually using `alertDescription()` a couple of lines below too, so I think it's a little better to have it as a separate function.

**Comment by spadgett** [comment]
jenkins rebuild

**Comment by kyoto** [comment]
jenkins rebuild

**Comment by spadgett** [comment]
jenkins rebuild

**Comment by spadgett** [comment]
prow status is missing from this PR. I'm not sure why, but we might need to green button merge when tests pass.

**Comment by kyoto** [comment]
@spadgett I had to rebase. Could you LGTM again?

**Comment by kyoto** [comment]
jenkins rebuild

**Comment by kyoto** [comment]
jenkins rebuild

**Comment by spadgett** [comment]
Hopefully #694 fixes the OLM failure

**Pull request merged**